config.py

```diff
@@ -8,17 +8,21 @@
                             help='faucet address')
 parser.add_argument('--private-key', type=str, nargs='?',
                             help='faucet private key')
+parser.add_argument('--block-count', type=int, nargs='?',
```

Comment:
Changed this to `int`.

Reply:
Looks good, I converted `block_count` to string (see comment below)

Reply:
Thanks!